resources/languages.ts

```diff
@@ -4,6 +4,11 @@ export type Lang = typeof languages[number];
 
 export type NonEnLang = Exclude<Lang, 'en'>;
 
+/**
+ * @deprecated remove this after we don't support `netRegexCn` style triggers
+ */
+export type LegacyLangSuffix = 'En' | 'De' | 'Fr' | 'Ja' | 'Cn' | 'Ko';
```

Comment:
Where does this get used?

Reply:
`regex${lang as LegacyLangSuffix}` `netRegex${lang as LegacyLangSuffix}`

Reply:
Sorry, I don't see what you're referring to.  Which file is that in?

Reply:
it's used to get legacy  `regexDe`/`netRegexDe` property in `popup-text.ts`

Reply:
```shell
$ git grep LegacyLangSuffix
resources/languages.ts:export type LegacyLangSuffix = 'En' | 'De' | 'Fr' | 'Ja' | 'Cn
' | 'Ko';
```

Reply:
Sorry for the confusion, I just don't see what you mean.  Did you mean to add these changes and they didn't save/commit?

Reply:
that's strange, I remembered I used it in `popup-test.ts`, maybe I lost it by mistake.